app/models/publication.rb

```diff
@@ -145,6 +145,12 @@ def self.preferred_versions
                                                  .where(%{NOT EXISTS (SELECT * FROM open_access_locations WHERE open_access_locations.publication_id = publications.id AND open_access_locations.source = '#{Source::SCHOLARSPHERE}')})
                                              }
   scope :activity_insight_oa_publication, -> { oa_publication.with_no_scholarsphere_oa_locations.joins(:activity_insight_oa_files).where.not(activity_insight_oa_files: { location: nil }) }
+  scope :troubleshooting_list, -> {
+    activity_insight_oa_publication
+      .where(%{(open_access_status != 'gold' AND open_access_status != 'hybrid') OR open_access_status IS NULL})
+      .includes(:activity_insight_oa_files)
+      .order('activity_insight_oa_files.created_at ASC')
```

Comment:
What does this do to the order of the list of publications when publications have more than one file? Does it matter?

Those aren't rhetorical questions - I don't know the answers. I just noticed that it doesn't look like any of the tests cover this scenario.

Other than that, I think this all looks good.

Reply:
So unfortunately if there are multiple files, it's ordered by the oldest one (which is why there isn't a test). I tried to get it to push any with a newer file to the bottom but couldn't figure it out. However, when I spoke to Ana, she said she was ok with that for the time being.